common/src/main/java/nl/tudelft/trustchain/common/DemoCommunity.kt

```diff
@@ -1,27 +1,92 @@
 package nl.tudelft.trustchain.common
 
+import android.content.Context
 import android.util.Log
+import com.frostwire.jlibtorrent.TorrentInfo
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.channels.BroadcastChannel
+import mu.KotlinLogging
 import nl.tudelft.ipv8.Community
 import nl.tudelft.ipv8.IPv4Address
+import nl.tudelft.ipv8.Overlay
 import nl.tudelft.ipv8.Peer
 import nl.tudelft.ipv8.android.IPv8Android
 import nl.tudelft.ipv8.attestation.trustchain.TrustChainCommunity
+import nl.tudelft.ipv8.keyvault.PrivateKey
 import nl.tudelft.ipv8.messaging.Address
 import nl.tudelft.ipv8.messaging.Deserializable
 import nl.tudelft.ipv8.messaging.Packet
+import nl.tudelft.ipv8.messaging.eva.TransferException
+import nl.tudelft.ipv8.messaging.eva.TransferProgress
 import nl.tudelft.ipv8.messaging.payload.IntroductionResponsePayload
 import nl.tudelft.ipv8.messaging.payload.PuncturePayload
+import nl.tudelft.trustchain.common.freedomOfComputing.AppPayload
+import nl.tudelft.trustchain.common.freedomOfComputing.AppRequestPayload
+import java.io.File
+import java.io.FileOutputStream
+import java.lang.Exception
 import java.util.*
 
-class DemoCommunity : Community() {
+private val logger = KotlinLogging.logger {}
+
+@Suppress("deprecation")
```

Comment:
What deprecation are you suppressing here and why?

Reply:
we aren't suppressing any, thanks for pointing that out, will remove it